### DISCUSSION TITLE
Support is dead, why Xiaomi manager Diyun Xiang does not care about CyberDog?

### DISCUSSION THREAD

**Discussion by MAVProxyUser** [MiRoboticsLab/cyberdog_ros2#160] [Q&A]
email to: xiangdiyun@xiaomi.com
Many developers are confused on why source is really closed, not open? 

Motion controller source is closed

Android App source is closed. 

URDF for Gazebo is missing!? 

Why is the dog not meant for developers? 

Many developers are seeking help, but can not get it. Please help. 

Why Diyun not caring about Cyberdog success? 
https://www.linkedin.com/in/diyun-xiang-6a492330/

**Comment by emarin44** [comment]
Use the force
